src/datumaro/plugins/data_formats/kaggle/base.py

```diff
@@ -42,71 +47,135 @@ def __init__(
         super().__init__(ctx=ctx)
 
         self._subset = subset
-
         self._path = path
-        self._columns = columns
 
-        self._items, label_cat = self._load_items(ann_file, columns)
-        self._categories = {AnnotationType.label: label_cat}
+        if "media" not in columns:
+            raise MissingFieldError("media")
+
+        self._label_cat = LabelCategories()
+        self._items = self._load_items(ann_file, columns)
+        self._categories = {AnnotationType.label: self._label_cat}
+
+    def _get_media_path(self, media_name: str):
+        if not media_name.lower().endswith(tuple(IMAGE_EXTENSIONS)):
+            for ext in IMAGE_EXTENSIONS:
+                media_path = osp.join(self._path, media_name + ext)
+                if osp.exists(media_path):
+                    return media_path
+            return None
+        else:
+            return osp.join(self._path, media_name)
+
+    def _parse_bbox_coords(self, bbox_str):
+        bbox_str = bbox_str.strip()
+
+        # Check if the string starts with "[" and ends with "]"
+        if bbox_str.startswith("[") and bbox_str.endswith("]"):
+            bbox_str = bbox_str[1:-1]
+
+        coords = list(filter(None, re.split(r"\s|,", bbox_str)))
+
+        # expected to output [x1, y1, x2, y2]
+        return [float(coord.strip()) for coord in coords]
```

Comment:
```suggestion
    def _parse_bbox_coords(self, bbox_str):
        coords = re.findall(r"[-+]?\d*\.\d+|\d+", bbox_str)

        if len(coords) != 4:
            raise ValueError("Bounding box coordinates must have exactly 4 values.")

        return [float(coord) for coord in coords]
```
It appears to be a function whose function is to extract coordinate values from a string and return them as a list of floats. I think it would be better to use a regular expression as above.

Reply:
Great, fixed at fc8b3ed6b7f04414afa45cf0219e81343bd9e61d